benchmarks/ridge/datasets/simulated_blobs.py

```diff
@@ -0,0 +1,41 @@
+from benchopt import BaseDataset, safe_import_context
+from benchopt.datasets import make_correlated_data
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+
+
+class Dataset(BaseDataset):
+    name = "Simulated_correlated_data"
+
+    parameters = {
+        "n_samples, n_features": [
+            (5_000_000, 100),
+            (5000, 5000),
```

Comment:
Actually it would be even more interesting to replace that setting by:

```suggestion
            (5_000, 10_000),
```

and measure performance when `alpha` is small.